lib/gather-fragment-imports.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 
-const inlineImports = require('@graphql-fragment-import/lib/inline-imports');
```

Comment:
do we need this change?

Reply:
So, when I ran `yarn test` from the top level of the project, it actually ran the linter and that linter complained about this line. :) Not a big deal but I imagine a project referring to itself this way is pretty uncommon.

Reply:
Interesting, Feel free to make the change then.

Reply:
With the existing code, the lint error that fails `yarn test` or `yarn lint` is:
```
"@graphql-fragment-import/lib" is extraneous.eslint[node/no-extraneous-require](https://github.com/mysticatea/eslint-plugin-node/blob/v11.1.0/docs/rules/no-extraneous-require.md)
```
so I figured I'd clean this up by using a relative path